README.md

```diff
@@ -1,2 +1,72 @@
 # Rerun ROS bridge
 A bridge for connecting Rerun with ROS 2 written in Rust.
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Building the Bridge](#building-the-bridge)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Introduction
+
+The Rerun ROS Bridge is a tool designed to facilitate communication between the Rerun framework and ROS 2 systems. This bridge is implemented in Rust.
+
+## Building the Bridge
+
+The bridge uses Pixi to install the dependencies and set up the environment. Please check out the following steps to build the bridge:
+
+1. Clone the repository:
+    ```sh
+    git clone https://github.com/rerun-io/rerun-ros.git
+    cd rerun-ros
+    ```
+
+2. Install Pixi:
+    ```sh
+    curl -fsSL https://pixi.sh/install.sh | bash
+    ```
+
+3. Use Pixi to install dependencies and set up the environment:
+    ```sh
+    pixi install
+    ```
+
+4. Build the project:
+    ```sh
+    pixi run build
+    ```
+
+## Usage
+
+To use the Rerun ROS Bridge, follow these steps:
+
+1. Run the bridge:
+    ```sh
+    pixi run ros2 rerun_ros rerun_ros -c config/example.toml
```

Comment:
This line appears to be missing the `run` sub-command in ros2:
```suggestion
    pixi run ros2 run rerun_ros rerun_ros -c config/example.toml
```

However, even after fixing that it appears to just exit immediately:
```
Starting bridge
Subscribing to topic: topic/foo with type: std_msgs/msg/String
Subscribing to topic: topic/bar with type: std_msgs/msg/Int32
```

We should include all the steps to get to an operational visualizer. When I finish running the example steps there should be a running rerun visualizer showing some kind of output produced by ROS.

Reply:
Ah, yeah, I didn't rebase this on top of `connect-rerun`, that's why you're not seeing much. I've rebased this on top of #12 and you should be seeing the following:

```console
$ pixi run ros2 run rerun_ros rerun_ros -c config/example.toml 
Starting bridge
Subscribing to topic: topic/bar and frame_id None with type: std_msgs/msg/Int32
Subscribing to topic: topic/foo and frame_id None with type: std_msgs/msg/String
Subscribing to topic: topic_stamped and frame_id frame5 with type: geometry_msgs/msg/TransformStamped
Subscribing to topic: topic and frame_id frame2 with type: geometry_msgs/msg/Transform
```

and the node should connect to the rerun viewer.